warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java

```diff
@@ -67,6 +67,14 @@ public static void setPropertyIfNecessary(String tableName, String propertyName,
         }
     }
 
+    public static void disableVersioningIfNecessary(TableOperations tops, Logger log) throws AccumuloException, AccumuloSecurityException {
```

Comment:
The log argument appears to be unused. I'd remove it if we don't need it.

Reply:
Thanks, will do